app/views/home/index.html.erb

```diff
@@ -7,18 +7,13 @@
 
 		<!--Welcome section-->
         <section class="waypoint video-block fullscreen" data-hash="welcome">
-			<%= video_tag(
-				"home/woman-small.mov",
-				id: "womanvideo",
-				autoplay: "autoplay",
-				muted: "muted",
-				loop: "loop",
-				poster: "home/woman_front.jpg",
-				class: "background-video",
-				"x-webkit-airplay": true,
-				"webkit-playsinline": true,
-				preload: "auto"
-			) %>
+
+			<video id="womanvideo" autoplay="autoplay" muted="muted" loop="loop" poster    ="assets/home/woman_front.jpg" 
+			 	    class="background-video" x-webkit-airplay="true" webkit-playsinline="true" preload="auto">
+					<source data-src="assets/home/woman-small.mov" type="video/mp4">
+					<source data-src="assets/home/woman-small.ogm" type="video/ogm">
+			</video>
+
```

Comment:
The idea is that the video is not loaded util the javascript runs, on QQ and other browsers.
Specifically, 'data-src' is invalid property attribute for video tag in html5, then the javascript sets the correct attribute 'src' if it is desktop. 
If to use pipeline, the video will be always loaded first, on QQ browser engine. 
My understanding is that not using pipeline hurts performance.